### PR TITLE
Style Landing, BeforeYouBegin, Affirmation (mobile)

### PR DESCRIPTION
**Pull request by alexchoiweb**
-Made several spacing changes according to design specs
  -used a mix of inline styling and `makeStyles()` to replace some `adjacent-margin` classes that weren't spacing 100% correctly. Wasn't able to pick which was more elegant, let me know if you have a preference or, even a better solution
-Record clearance logo now acts as home button, redirecting user back to Landing
-Fixed Navbar links spacing on mobile (links were previously stuck to bottom without margin on mobile)

https://user-images.githubusercontent.com/35116846/113470582-82816000-940b-11eb-8fe7-fc35bc832d0a.mov

### CODE REVIEW

File: react/client/src/components/Header.tsx [hunk 1]
```diff
@@ -61,7 +61,9 @@ const Header = ({ pageNumber }: HeaderProps) => {
 
   return (
     <div className={`${classes.root} app-header`}>
-      <img src={icon} alt="" />
+      <Link to="/">
+        <img src={icon} alt="" />
+      </Link>
```
Comment (aahvocado) [inline react/client/src/components/Header.tsx:66]: Yeah, good catch on the header logo not being a link.

File: react/client/src/styles/utilityStyles.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ const useUtilityStyles = makeStyles((theme) =>
       maxWidth: '850px',
       marginLeft: 'auto',
       marginRight: 'auto',
+      marginTop: '49px',
```
Comment (aahvocado) [inline react/client/src/styles/utilityStyles.ts:19]: This is such an arbitrary value (though I suspect that's what the design has) but I'm good with it. We would want to use more common numbers that the entire app follows but this is so low priority that we can always come back to it.

**Review by aahvocado — APPROVED** [review]
I am fine and in fact support the change from classnames to makeStyles.

File: react/client/src/styles/utilityStyles.ts (hunk 2 above)
Comment (alexchoiweb) [reply react/client/src/styles/utilityStyles.ts:19]: Yep you are right, I followed what the design has right now.
Yep that sounds good about the common numbers. Thanks for reviewing!

**Pull request merged**